Resources/Public/less/components.css

```diff
@@ -102,6 +102,18 @@
 .flag-icon-de.flag-icon-squared {
   background-image: url(flags/1x1/de.svg);
 }
+.flag-icon-en {
```

Comment:
Do not change files in `CSS` and `LESS` folders. Those folders are generating automatically  `theme_t3kit/felayout_t3kit`
https://github.com/t3kit/theme_t3kit/blob/master/Resources/Public/README.md
Use `theme_t3kit/felayout_t3kit/dev/styles/components.less`